.env.example

```diff
@@ -0,0 +1,21 @@
+PORT=8081
+
+DB_HOST=localhost
+DB_PORT=5432
+DB_USERNAME=postgres
+DB_PASSWORD=Sherman33
```

Comment:
I know it is just a local training database, but do not commit passwords :sweat_smile: